litmus/PTX/SyncBar4.litmus

```diff
@@ -3,12 +3,13 @@ PTX Bar
 {
 x=0;
 y=0;
-P0:r0=0;
-P1:r2=0;
+P0:r0=0; P0:r2=0;
+P1:r1=0; P1:r3=0;
 }
  P0@cta 0,gpu 0         | P1@cta 0,gpu 0         ;
  st.weak x, 1           | st.weak y, 2           ;
- bar.cta.arrive 0       | bar.cta.sync 0         ;
+ ld.weak r1, 1          |                        ;
```

Comment:
You don't need this store, you can simply give a different initial value to the register